crates/ty_server/src/session.rs

```diff
@@ -98,6 +97,8 @@ pub(crate) struct ProjectState {
     /// for so that we can clear the diagnostics for all of them before we go
     /// to update any of them.
     pub(crate) untracked_files_with_pushed_diagnostics: Vec<Url>,
+
+    pub(crate) db: ProjectDatabase,
```

Comment:
This change isn't necessary but it hopefully avoids similar issues in the future